src/Tools/GenerateSdkPackages/make-all.ps1

```diff
@@ -0,0 +1,65 @@
+Param(
+    [string]$version = "26127.00",
+    [string]$branch = "d15rel",
+    [string]$outPath = $null,
+    [string]$fakeSign = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try {
+    if ($outPath -eq "") {
```

Comment:
Dumb question: why not just use https://blogs.technet.microsoft.com/heyscriptingguy/2011/05/22/use-powershell-to-make-mandatory-parameters/?